protocols/1602-part2/adaptor_ligation.ot2.py

```diff
@@ -0,0 +1,182 @@
+from opentrons import labware, instruments, modules, robot
+
+metadata = {
+    'protocolName': 'End Prep of cDNA Library, Adaptor Ligation',
+    'author': 'Nick <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request'
+}
+
+# labware and modules
+magdeck = modules.load('magdeck', '1')
+mag_plate = labware.load('biorad-hardshell-96-PCR', '1', share=True)
+trough = labware.load('trough-12row', '2')
+new_plate = labware.load('biorad-hardshell-96-PCR', '3', 'fresh plate')
+
+tipracks_m10 = [labware.load('tiprack-10ul', slot)
+                for slot in ['9', '10', '11']]
+tipracks_m300 = [labware.load('tiprack-200ul', slot)
+                 for slot in ['4', '5', '6', '7', '8']]
+
+# pipette setup
+m10 = instruments.P10_Multi(
+    mount='left',
+    tip_racks=tipracks_m10)
+
+m300 = instruments.P300_Multi(
+    mount='right',
+    tip_racks=tipracks_m300)
+
+# reagent setup
+end_prep_reaction_buffer = trough.wells('A1')
+end_prep_enzyme = trough.wells('A2')
+diluted_adapter = trough.wells('A3')
+ligation_enhancer = trough.wells('A4')
+ligation_mastermix = trough.wells('A5')
+USER_enzyme = trough.wells('A6')
+oligo_beads = trough.wells('A7')
+ethanol = trough.wells('A8')
+TE_buffer = trough.wells('A9')
+
+
+def run_custom_protocol(
+        sample_columns_separated_by_commas: str = '1,2,3,4,5,6,7,8,9,10,11,12'
+):
+
+    m300_tip_count = 0
+    m10_tip_count = 0
+
+    acceptable_cols = [str(col) for col in range(1, 13)]
+    cols = [col.strip()
```

Comment:
If you convert each item in the `cols` list to int, then you can just check each item against `<13` and `>0` rather than comparing them to a list of string of numbers